vendor/bat-native-ads/data/test/invalid_model.json

```diff
@@ -0,0 +1,1546 @@
+Invalid JSON
```

Comment:
Please be more verbose, i.e. `ml/models/invalid_spam_text_classification_model.json`. Please rename other json models too

Reply:
Done